python/igraphalg.py

```diff
@@ -1,23 +1,20 @@
-from collections import defaultdict
-from typing import Dict, List, Tuple
-
-import igraph
+from typing import List
 
 import mgp
 
+from mgp_igraph import MemgraphIgraph
+
 
 @mgp.read_proc
-def get_flow(
+def maxflow(
     ctx: mgp.ProcCtx,
-    start_v: mgp.Vertex,
-    end_v: mgp.Vertex,
-    edge_property: str = "weight",
+    source: mgp.Vertex,
+    destination: mgp.Vertex,
```

Comment:
This parameter is called `target` in igraph ([reference](https://igraph.org/python/api/latest/igraph.Graph.html#maxflow)).